### PR TITLE
Upgrade to sort orders so the text can be changed from locales

### PR DESCRIPTION
**Pull request by ErwannNevou**
This change makes this possible : 

```js
$.fn.bootstrapTable.locales['fr-FR'] = {
    [ ... ]
    sortOrders: function () {
      return {
        asc: 'Croissant',
        desc: 'Décroissant'
      }
    },
    [ ... ]
 }
```

wich is useful :+1:

### CODE REVIEW

**Comment by djhvscf** [comment]
Can you provide an example?

**Comment by ErwannNevou** [comment]
Hello,

Here is an example (All changes are in the multi-sort popup): 
First without my changes :  [No changes](https://jsfiddle.net/u2ta4utm/6/)
In this, we can't translate 'Ascending' and 'Descending' without changing the code in bootstrap-table-multiple-sort.js which is annoying. 

But I wanted to be able to translate it from the locales like the other strings in the popup.

Second with my changes : [With changes](https://jsfiddle.net/34tp1htj/3/) 
In this one, we can change the translation of 'Ascending' and 'Descending' without changing the code in bootstrap-table-multiple-sort.js. So we can change all the strings of the popup from the locales.

The multi sort still works, I just changed the translation 

PS : The translatons are in French

**Comment by djhvscf** [comment]
Thanks! Lools good

File: src/extensions/multiple-sort/bootstrap-table-multiple-sort.js [hunk 1]
```diff
@@ -203,6 +199,12 @@
         },
         formatDuplicateAlertDescription: function() {
             return "Please remove or change any duplicate column.";
+        },
+        sortOrders : function () {
+            return {
+              asc: 'Ascending',
+              desc: 'Descending'
+            }
```
Comment (wenzhixin) [inline src/extensions/multiple-sort/bootstrap-table-multiple-sort.js]:
```
return {
    asc: 'Ascending',
    desc: 'Descending'
};
```

File: src/extensions/multiple-sort/bootstrap-table-multiple-sort.js [hunk 2]
```diff
@@ -203,6 +199,12 @@
         },
         formatDuplicateAlertDescription: function() {
             return "Please remove or change any duplicate column.";
+        },
+        sortOrders : function () {
```
Comment (wenzhixin) [inline src/extensions/multiple-sort/bootstrap-table-multiple-sort.js]: It's better to use `formatSortOrders`

**Comment by wenzhixin** [comment]
Please help to update the README

**Comment by ErwannNevou** [comment]
Done

**Comment by wenzhixin** [comment]
Thanks!

**Pull request merged**